tv_apps/fling-player/js/fling_player.js

```diff
@@ -124,13 +127,30 @@
     this.moveTimeBar('buffered', 0);
     this.writeTimeInfo('elapsed', 0);
     this.writeTimeInfo('duration', 0);
-    this.setPlayButtonState('pause');
+    this.setPlayButtonState('paused');
   };
 
   proto.showLoading = function (loading) {
     this._loadingUI.hidden = !loading;
   };
 
+  proto.showInitialMessage = function (bool) {
+    if (bool) {
```

Comment:
Since you just wrote a 'bool' here, what does this condition really trying to do?